.buildkite/pipeline.yml

```diff
@@ -35,8 +35,10 @@ steps:
         run: app
         workdir: /go/src/github.com/m3db/m3
     <<: *common
-  - name: "Services, Tools, Metalint"
-    command: make clean install-vendor-m3 services tools metalint
```

Comment:
Are `services` and `tools` not relevant any more?

Reply:
👍